torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -98,6 +98,45 @@ class TORCH_CUDA_CU_API ARangeOp : public Expr {
   Val* linear_index_ = nullptr;
 };
 
+class TORCH_CUDA_CU_API EyeOp : public Expr {
```

Comment:
nit: I know this op is self-explanatory given a tensor library context but some comments on this Op would be helpful.

Reply:
added